opentelemetry-aws/CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 
 - reduce `tokio` feature requirements #750
 - Update to opentelemetry v0.18.0
+- Move Xray IdGenerator from `opentelemetry-rust` to `opentelemetry-aws` [#33](https://github.com/open-telemetry/opentelemetry-rust-contrib/pull/33)
```

Comment:
the entry got into the wrong place. Could you fix that in a follow up?

Reply:
created #38